README.md

```diff
@@ -58,12 +58,29 @@ npm start
 
 ### Docker
 You need to have [Docker](https://docs.docker.com/install/) and [docker-compose](https://docs.docker.com/compose/install/) installed.
+Clone the repo:
 ```bash
 git clone https://github.com/poetapp/node.git
 cd node
+```
+
+To start the Po.et node environment, run:
+```bash
 docker-compose up --build
 ```
 
+Using the instructions above, new blocks have to be generated manually.
+This is often desirable during development. To have blocks automatically
+generated, follow the instructions below instead to mine a new block
+every 5 seconds.
+
+```bash
+  $ cp docker-compose.override.yml.example docker-compose.override.yml
+  # Edit docker-compose.override.yml and uncomment
+  # the `regtest-watcher` service in `docker-compose.yml`
+  $ docker-compose up --build
```

Comment:
I get this error here:

```
╰─$ dc up --build
Creating network "node_default" with the default driver
Creating volume "node_db" with local driver
Creating volume "node_data" with local driver
Pulling mongo (mongo:3.4)...
3.4: Pulling from library/mongo
795df959b1e6: Pull complete
a2ed688142f5: Pull complete
18570f9e34b4: Pull complete
3f5dc7b38863: Pull complete
4b6b976af737: Pull complete
f9a7b8ce28cf: Pull complete
196058143fb4: Pull complete
52795aba6f28: Pull complete
818a78303399: Pull complete
bda7c373cb53: Pull complete
9ba65ea459cd: Pull complete
Digest: sha256:5bd63fb222201a5d69513f028a8c932a3801cdc939640396292e8ec7ed5a3c82
Status: Downloaded newer image for mongo:3.4
Pulling regtest-watcher (node_poet-node:latest)...
ERROR: The image for the service you're trying to recreate has been removed. If you continue, volume data could be lost. Consider backing up your data before continuing.

Continue with the new image? [yN]y
Pulling regtest-watcher (node_poet-node:latest)...
ERROR: pull access denied for node_poet-node, repository does not exist or may require 'docker login'
```

Reply:
The docker image being used is incorrect, or you have not built the image locally and named it `node_poet-node:latest` - this looks like the container name that docker-compose creates when running `docker-compose up`. You will either need to prebuild node, with a specific name and tag, or reference the image digest locally (which could possibly be unique for each user), or pull `poetapp/node:latest` or `poetapp/node:v1.8.2` or any other tag here. Since its a public image, you will not need to login to pull the image)

Reply:
OK. I added `poetapp` to the container name and it is building now.

Reply:
Now I am getting this:

```
Status: Downloaded newer image for poetapp/node:latest
Creating node_bitcoind-1_1 ... done
Creating node_mongo_1      ... done
Creating node_ipfs_1       ... done
Creating node_rabbit_1     ... done
Creating node_regtest-watcher_1 ... error
Creating node_poet-node_1       ... done
Creating node_bitcoind-2_1      ... done

ERROR: for node_regtest-watcher_1  Cannot start service regtest-watcher: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"scripts/regtest-watcher.js\": stat scripts/regtest-watcher.js: no such file or directory": unknown

ERROR: for regtest-watcher  Cannot start service regtest-watcher: OCI runtime create failed: container_linux.go:348: starting container process caused "exec: \"scripts/regtest-watcher.js\": stat scripts/regtest-watcher.js: no such file or directory": unknown
ERROR: Encountered errors while bringing up the project.
```

Reply:
eugh... yeah obv the regest-watcher script wont exist in the latest or v1.8.2 tags.

change it to this: `poetapp/node:v-6e882cd-beta`

This is the Docker image tag built for this PR's highest commit